src/Inline/Element/AbstractInline.php

```diff
@@ -59,4 +59,15 @@ public function setParent($parent)
 
         return $this;
     }
+
+    /**
+     * @param string     $key
+     * @param mixed|null $default
```

Comment:
Good point.  I think I'm using that elsewhere, so I'll replace all such instances.